pkgs/cosmic-comp/package.nix

```diff
@@ -61,7 +61,6 @@ rustPlatform.buildRustPackage {
 
   postInstall = ''
     mkdir -p $out/etc/cosmic-comp
-    cp config.ron $out/etc/cosmic-comp/config.ron
```

Comment:
Unless it's changed since I looked the week before last, there is still another file to copy instead. But also until cosmic-comp is upgraded past that commit, we do not want this since it'll break all default keybinds

I've gone ahead and merged the other parts of this PR to main